nodes/Arguments.py

```diff
@@ -1,2 +1,25 @@
+"""
+Arguments astroid node
+
+Attributes:
+    - args         (list)  list of non-keyword argument names
+    - vararg       (arg)   variable-length argument or None
+    - kwonlyargs   (list)  list of keyword-only argument names
+    - kwarg        (arg)   single arg nodes and keyword only arguments
+    - defaults     (list)  n-tuple of the default values of the last n
```

Comment:
Use the type `Tuple` to differentiate from `List`; also, if a value can be `None`, indicate this with a type like `Tuple | None`